requirements/dev.txt

```diff
@@ -3,5 +3,5 @@
 
 django-debug-toolbar
 sphinx
-django>=1.4,<1.10
```

Comment:
@grokcode Deps in `dev.txt` don't get installed when you `pip install django-fitbit` so there is no risk of accidentally upgrading django there, is that what you mean?